oak_functions/client/android/com/google/oak/functions/android/client/MainActivity.java

```diff
@@ -65,9 +65,7 @@ public void onCreate(Bundle savedInstanceState) {
 
     // Set default URI of the Oak Functions application.
     EditText uriInput = findViewById(R.id.uriInput);
-    // 10.0.2.2 is routed to the host machine by the Android emulator.
-    // TODO(#2172): Connect to API Gateway instead of the host machine.
-    uriInput.setText("10.0.2.2:8080");
+    uriInput.setText("https://weather-lookup-endpoint-62sa4xcfia-nw.a.run.app");
```

Comment:
Could you add a comment / link on where this URL comes from?

Reply:
Done